config/sync/core.extension.yml

```diff
@@ -33,6 +33,7 @@ module:
   datetime: 0
   datetime_range: 0
   default_content: 0
+  diff: 0
```

Comment:
Is diff needed or is this just a nice to have?

Reply:
just nice to have. i noticed we had the module in composer so i figured we could enable it.